packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -276,6 +277,47 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to null. If null, only the text field is updated.
   final ValueChanged<T?>? onSelected;
 
+  /// Defines the keyboard focus for this widget.
+  ///
+  /// The [focusNode] is a long-lived object that's typically managed by a
+  /// [StatefulWidget] parent. See [FocusNode] for more information.
+  ///
+  /// To give the keyboard focus to this widget, provide a [focusNode] and then
+  /// use the current [FocusScope] to request the focus:
+  ///
+  /// ```dart
+  /// FocusScope.of(context).requestFocus(myFocusNode);
+  /// ```
```

Comment:
The analyzer is probably failing due to missing `{@tool snippet}`.

Reply:
And maybe we can create a `{@macro ...}` for this documentation. Seems it is the same as the `TextField.focusNode`:)

Reply:
I see. That makes sense:)